Lib/collections/__init__.py

```diff
@@ -1207,12 +1207,9 @@ def count(self, sub, start=0, end=_sys.maxsize):
         if isinstance(sub, UserString):
             sub = sub.data
         return self.data.count(sub, start, end)
-    def encode(self, encoding=None, errors=None): # XXX improve this?
-        if encoding:
-            if errors:
-                return self.__class__(self.data.encode(encoding, errors))
-            return self.__class__(self.data.encode(encoding))
-        return self.__class__(self.data.encode())
+    def encode(self, encoding='utf-8', errors='strict'):
+        encoding, errors = (encoding or 'utf-8'), (errors or 'strict')
```

Comment:
The case ``encoding=''`` is different from ``encoding=None``.  Expand this code to:

	encode = 'utf-8' if encoding is None else encoding
	errors = 'strict' if errors is None else errors